### DISCUSSION TITLE
Sida för kurslitteratur och gråttanbiblioteket

### DISCUSSION THREAD

**Discussion by CL-Webbunderhallare** [Webbutvecklare-CL/cl-sektionen.se#143] [Ideas]
En sida där man kan lägga upp och sälja böcker. Kanske ha ett google spredsheet som läses in.

Gråttanbiblioteket kan också bara vara ett google spredsheet som läses in.

**Comment by Emeliehanna11** [comment]
Jaa! snälla gör så man kan sortera på kurser, böcker, och inriktningar <3 google sheets dokumentet är lite jobbigt att man måste scrolla så mycket och det är svårt att hitta för alumner/äldre studenter som har böcker att sälja.
